extensions/timers/01_timers.js

```diff
@@ -480,7 +498,7 @@
     let callback;
 
     if ("function" === typeof cb) {
-      callback = Function.prototype.bind.call(cb, globalThis, ...args);
+      callback = FunctionPrototypeBind(cb, globalThis, ...args);
```

Comment:
@lucacasonato 
I'm not too familiar or sure about this, but the tests seem to pass